style/sass/README.md

```diff
@@ -38,8 +38,8 @@ Selectors
 * Use meaningful names: `$visual-grid-color` not `$color` or `$vslgrd-clr`.
 * Use ID and class names that are as short as possible but as long as necessary.
 * Avoid using the direct descendant selector `>`.
-* Avoid nesting more than 4 selectors deep.
-* Don't nest more than 6 selectors deep.
+* Avoid nesting more than 3 selectors deep.
+* Don't nest more than 5 selectors deep.
```

Comment:
+1 - I think we can remove this as the previous line sets the recommended limit.